test/test_api/test_results_manager.py

```diff
@@ -88,8 +113,43 @@ def _check_metric_dict(metric_dict, status_types):
     for key, vals in metric_dict.items():
         # ^ is a XOR operator
         # True and False / False and True must be fulfilled
-        assert all([(s == STATUS2MSG[StatusType.SUCCESS]) ^ isnan
-                    for s, isnan in zip(status_types, np.isnan(vals))])
+        assert all([(s == StatusType.SUCCESS) ^ np.isclose([val], [worst_val])
+                    for s, val in zip(status_types, vals)])
+
+
+def _check_metric_results(scores, metric, run_history, ensemble_performance_history):
+    if metric.name == 'accuracy':
```

Comment:
could you explain the need for this condition?

Reply:
I added the doc string.
But just in case I will explain.
In this case, we are interested in `accuracy`, but ensemble history has only `log_loss` as hard-coded.
In this case, ensemble results should be None and merged data also should include the same data as the data obtained from `run_history`.

Reply:
okay. Thanks